### PR TITLE
feature(auth): add Masa.BuildingBlocks.BasicAbility.Auth.Contracts project

### PR DESCRIPTION
**Pull request by 15168440402**

### CODE REVIEW

File: src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Auth.Contracts/_import.cs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+global using Masa.BuildingBlocks.BasicAbility.Auth.Contracts.Enum;
```
Comment (zhenlei520) [inline src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Auth.Contracts/_import.cs:4]: Change the file name to `_Imports.cs`

File: src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Auth/_import.cs [hunk 2]
```diff
@@ -1,6 +1,5 @@
 // Copyright (c) MASA Stack All rights reserved.
 // Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 
-global using Masa.BuildingBlocks.BasicAbility.Auth.Enum;
```
Comment (zhenlei520) [inline src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Auth/_import.cs:4]: Change the file name to `_Imports.cs`

**Review by zhenlei520 — APPROVED** [review]
@doddgu

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**